blueprints/app/files/bower.json

```diff
@@ -1,18 +1,15 @@
 {
   "name": "<%= name %>",
   "dependencies": {
-    "ember": "2.1.0",
+    "ember": "1.13.10",
     "ember-cli-shims": "0.0.6",
     "ember-cli-test-loader": "0.2.1",
-    "ember-data": "2.1.0",
+    "ember-data": "1.13.14",
     "ember-load-initializers": "0.1.7",
     "ember-qunit": "0.4.13",
```

Comment:
Can you bump this to 0.4.15?  We are are currently using the the `wait` helper, which was added in 0.4.14.